references/cli/def.go

```diff
@@ -1168,7 +1169,9 @@ func NewDefinitionGenCUECommand(_ common.Args) *cobra.Command {
 			"# Generate CUE schema for provider type with custom types\n" +
 			"> vela def gen-cue -t provider --types *k8s.io/apimachinery/pkg/apis/meta/v1/unstructured.Unstructured=ellipsis /path/to/myprovider.go\n" +
 			"# Generate CUE schema for provider type with custom output path\n" +
-			"> vela def gen-cue -t provider -o /path/to/myprovider.cue /path/to/myprovider.go\n",
+			"> vela def gen-cue -t provider -o /path/to/myprovider.cue /path/to/myprovider.go\n" +
+			"# Output CUE schema to STDOUT\n" +
+			"> vela def gen-cue -t provider -o - /path/to/myprovider.go",
```

Comment:
Do we need this `-o` flag? Maybe we can just output the result to stdout, if the user want to write to a file, he can use `vela def gen-cue -t provider > mycue.cue`

Reply:
WDYT? @Somefive

Reply:
hmm, do we need to consider Windows users?

Reply:
well, my bad, `>` is also supported in powershell

Reply:
Do we need this: `Output CUE file path, if not specified, the CUE file will be generated in the same directory`? if not, only stdout +1

Reply:
Updated